src/app/core/building/building.component.html

```diff
@@ -89,7 +89,7 @@ <h3 class='red'>※ 入力必須項目</h3>
               <mat-form-field class="full-width-input" appearance='outline'>
                 <mat-label>防火地域</mat-label>
                 <mat-select [(ngModel)]='blueprint.fireZone' id="fireZone" name="fireZone" #fireZone="ngModel" required>
-                  <mat-option value="null">防火地域</mat-option>
```

Comment:
@hirao-k Is blueprint analysis execution API already supported **unspecified** for fire zone?

Reply:
@thanhtetaung   I haven't.  After I deploy, I'll tell you that.

Reply:
@thanhtetaung  I deployed it to API.

Reply:
@hirao-k thanks.